src/content/docs/infrastructure/host-integrations/host-integrations-list/stripe-integration.mdx

```diff
@@ -0,0 +1,123 @@
+
+To proceed further, you will need a New Relic license key. See [New Relic API Keys](/docs/apis/intro-apis/new-relic-api-keys/) to get your license key.
+On a successful stripe payment, we receive a JSON from the stripe API service. We need to send these values to New Relic by formatting the JSON as per New Relic Metric API requirements.
+
+Below is the javascript code snippet which formats the data and sends it to New Relic metric API URL
+```javascript
+const sendDataToNewRelic = (sessionJSON) => {
+  const currentDate = new Date();
+  const currentTime = currentDate.getTime();
+  const nrMetrics = [{
+    "metrics": [
+      {
+        "name": "stripe_metrics",
+        "timestamp": currentTime,
+        "value": 0,
+        "attributes": {
+          "stripe.session_id": sessionJSON.id,
+          "stripe.amount_total": sessionJSON.amount_total,
+          "stripe.customer_email": sessionJSON.customer_details.email,
+          "stripe.created_at": sessionJSON.created,
+          "stripe.currency" : sessionJSON.currency,
+          "stripe.payment_status": sessionJSON.payment_status,
+          "stripe.status": sessionJSON.status,
+          "stripe.discount_amount":sessionJSON.total_details.amount_discount,
+          "stripe.shipping_amount":sessionJSON.total_details.amount_shipping,
+          "stripe.tax_amount":sessionJSON.total_details.amount_tax
+        }
+      }
+    ]
+  }]
+
+  const nrMetricAPI = 'https://metric-api.newrelic.com/metric/v1'
+  axios({
+    url: nrMetricAPI,
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+      'Api-Key': 'LICENSE_KEY'
+     },
+    data: JSON.stringify(nrMetrics)
+  })
+   .then((response) => {
+    // Handle the response here if needed
+    console.log('Metrics data sent successfully:', response.data);
+  })
+  .catch((error) => {
+    // Handle any errors that occur during the request
+    console.error('Error sending metrics data to New Relic:', error);
+  })
+}
+
+```
+**Note:** Change the `sessionJSON` according to your response and call this function where you are getting Stripe metrics in your application.
+``````javascript
+ sendDataToNewRelic (session)
+```
+
+## Monitor your application
+You can choose our pre-built Stripe dashboard to monitor your application metrics. To set up this dashboard:
+1. Go to **[one.newrelic.com](https://one.newrelic.com/)** and click on **+ Add data**.
+2.  Click on the **Dashboards** tab.
+3.  In the search box, type `Stripe`.
+4.  When you see our pre-build dashboard, click on it to install it in your account.
+
+Once your application is integrated by following the above steps, the dashboard should display metrics.
+
+To instrument the Stripe quickstart and to see metrics and alerts, you can also follow our [Stripe quickstart page](https://newrelic.com/instant-observability/stripe) by clicking on the **Install now** button.
```

Comment:
```suggestion
## Monitor your application
You can choose our pre-built Stripe dashboard to monitor your application metrics. To set up this dashboard, go to [https://onenr.io/0ZQWYYE1rRW] and follow the instructions there. Once installed, the dashboard should display metrics.
```

Thanks @sjyothi54 . I replaced this section with just a single link to go to the UI.